src/core/sagas/autocomplete.ts

```diff
@@ -2,21 +2,66 @@ import * as effects from 'redux-saga/effects';
 import FluxCapacitor from '../../flux-capacitor';
 import Actions from '../actions';
 import Adapter from '../adapters/autocomplete';
+import Configuration from '../configuration';
 import Selectors from '../selectors';
 import Store from '../store';
+import { fetch } from '../utils';
 
 export namespace Tasks {
+  // tslint:disable-next-line max-line-length
   export function* fetchSuggestions(flux: FluxCapacitor, { payload: query }: Actions.FetchAutocompleteSuggestions) {
     try {
-      const field = yield effects.select(Selectors.autocompleteCategoryField);
-      const res = yield effects.call(
+      const state = yield effects.select();
+      const field = Selectors.autocompleteCategoryField(state);
+      const location = Selectors.location(state);
```

Comment:
It won't actually conflict due to scope, but should we be shadowing the global `location` object given that this is eventually going to be run in a browser?